setup.py

```diff
@@ -52,7 +52,7 @@
 DOWNLOAD_URL = "https://github.com/pydicom/pydicom/archive/master.zip"
 LICENSE = "MIT"
 VERSION = __version__
-# PACKAGE_DATA = {'pydicom': ['../tests/tests_files/*']}
+PACKAGE_DATA = {'pydicom': ['tests/test_files/*', 'tests/charset_files/*']}
 REQUIRES = []
```

Comment:
Ok, just a quick follow-up: what I really see as different is the kind of bugs that may appear. Practically all bugs in `pydicom` have to do with specific DICOM data (invalid data that nevertheless has to be parsed, or specific data that has not been tested), and it is highly unlikely IMHO that the existing tests will fail with them. So I see no real benefit to the test data being installed. But that is just my opinion...